.github/workflows/beam_PreCommit_Java_ElasticSearch_IO_Direct.yml

```diff
@@ -98,7 +98,6 @@ jobs:
         uses: ./.github/actions/gradle-command-self-hosted-action
         with:
           gradle-command: |
-            :sdks:java:io:elasticsearch-tests:elasticsearch-tests-5:build \
```

Comment:
Change on github action won't be effective in PR so the ElasticSearch IO Direct would fail due to target not found.